docs/versioned/admin/admin-overview.md

```diff
@@ -0,0 +1,66 @@
+
+For the Eventing component, you can configure Brokers that facilitate the routing and management of events. You can configure and process event data with Apache Kafka. You can also install and configure Istio for your traffic management and telemetry needs.
+
+| Concepts and Procedures | Reference |
+| --- | --- |
+| Install Istio for Knative<br>Supported Autoscaler Types<br>Configuring private Services<br>Configuring ingress class<br>Configuring certificate class<br>Configuring custom domains<br>Configuring HTTP<br>Tag resolution<br>Configuring Activator capacity<br>Exclude namespaces from the Knative webhook<br>Configuring high-availability components<br>Configuring the ingress gateway<br>Configuring domain names<br>Configuring Kafka features<br>Channel types and defaults<br>Configure Broker defaults<br>Configure Channel defaults<br>Configure Apache Kafka Channel defaults<br>Configure event source defaults<br>Configure Sugar Controller<br>Configure KEDA Autoscaling of Knative Kafka Resources<br>Knative reference mapping<br>Transport Encryption<br>Sender Identity<br>Eventing with Istio| Administrator configuration options<br>Configuring metrics<br>Configuring targets<br>Configuring scale to zero<br>Configuring concurrency<br>Configuring the requests per second (RPS) target<br>Configuring scale bounds<br>Additional autoscaling configuration for Knative Pod Autoscaler<br>Configure Deployment resources<br>Feature and extension flags<br>Configure the Defaults ConfigMap |
+
+## Monitoring and Observability
+
+Both the Serving and Eventing components are well maintained with logging, tracing, and metric collections.
+
+| Concepts and Procedures | Reference |
+| --- | --- |
+| Request traces<br>Collecting logs<br>Configuring logging<br>Configuring Request logging<br>Collecting metrics<br>Accessing CloudEvent traces<br>Collecting logs<br>Configuring logging<br>Collecting metrics | Service metrics<br>Metrics Reference |
+
+## Security
+
+Knative provides robust security and access control measures with the certificate manager, networking TLS certificates, domain and cluster encryption, and application security.
+
+| Concepts and Procedures | Reference |
+| --- | --- |
+| Install cert-manager <br>Using a custom TLS certificate for DomainMapping<br>Configure cert-manager integration<br>Configure external domain encryptio><br>Configure cluster-local domain encryption<br>Configure Knative system-internal encryption<br>Installing Security-Guard<br>Security-guard quickstart<br>Security-Guard example alerts<br>Transport Encryption<br>Sender Identity<br>Verifying Knative Images | Using extensions enabled by QPOptions<br>About Security-Guard<br> |
+
+## Updates and Maintenance
+
+In addition to updates, Knative provides several configurations for maintaining your clusters in optimal condition including resources, volume, traffic, and load balancing.
```

Comment:
Again, I'd emphasize integrating with existing components for things like resource limits and volumes (using the existing Kubernetes feature).

The "gradual rollout of traffic" and "developer configuration options" are probably more appropriate for the developer documentation (most of the site) than cluster administrators.  There *are* settings administrators can use to adjust the default behavior for users that don't specify behaviors, so you might have a setting on "adjusting cluster defaults".  That generally feels like it falls under "configuration", though.